src/Shouldly/ShouldlyMessage.cs

```diff
@@ -240,20 +240,21 @@ public string CreateExpectedErrorMessage()
         private static string CreateActualVsExpectedMessage(IShouldlyAssertionContext context)
         {
             var codePart = context.CodePart;
-            string message = string.Format(@"
-    {0}
-        {1}
-    {2}
-        but was
-    {3}",
-                codePart, context.ShouldMethod.PascalToSpaced(), context.Expected.ToStringAwesomely(), context.Actual.ToStringAwesomely());
+            var message = string.Format(@"
+            {0}
+                {1}
+            {2}
+                but was
+            {3}",
+                codePart, context.ShouldMethod.PascalToSpaced(), context.Expected.ToStringAwesomely(),
+                context.ShouldMethod == "ShouldNotBe" ? string.Empty : context.Actual.ToStringAwesomely());
```

Comment:
I'm not sure that Context.IsNegatedAssertion would work here. It returns true if the shouldMethod contains "Not" but we explicitly want to check for "ShouldNotBe". Maybe excent the context with a isShouldNotBe assertion? At some point, the explicit check will have to be done using a hard coded string, unless we introduce some kind of enum. Thoughts?